lib/testRunResult.spec.ts

```diff
@@ -0,0 +1,46 @@
+import TestRunResult from "./testRunResult";
+import { TestRunResponse, TestStatus } from "./types";
+
+describe("TestRunResult", () => {
+  it("only required images", () => {
+    const testRunResponse: TestRunResponse = {
+      url: "url",
+      status: TestStatus.ok,
+      pixelMisMatchCount: 12,
+      diffPercent: 0.12,
+      diffTollerancePercent: 0,
+      id: "some id",
+      imageName: "imageName",
+      merge: false,
+    };
+
+    const result = new TestRunResult(testRunResponse, "http://localhost");
+
+    expect(result.testRunResponse).toBe(testRunResponse);
+    expect(result.imageUrl).toBe("http://localhost/imageName");
+    expect(result.diffUrl).toBe(undefined);
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Visual-Regression-Tracker/sdk-js/pullRequest?prid=6318104)